examples/tutorials/Part 0 - Installation Guide.md

```diff
@@ -0,0 +1,228 @@
+
+# Part 0: Installing PySyft
+
+Before you start following the tutorials, you need to install PySyft on your computer.
+
+Here are the PySyft installation instructions for Linux, MacOs and Windows.
+
+Just scroll down to find your preferred OS:
```

Comment:
Maybe put links here to the sections of the document rather than saying to scroll down

Reply:
Good idea, done!